pkg/chain/local/tbtc.go

```diff
@@ -180,20 +180,11 @@ func (tlc *TBTCLocalChain) CreateDeposit(
 	keepAddress := generateAddress()
 	tlc.OpenKeep(keepAddress, common.HexToAddress(depositAddress), signers)
 
-	utxoValueBytesSlice, _ := hex.DecodeString(defaultUtxoValueHex)
-	var utxoValueBytes [8]byte
-	copy(utxoValueBytes[:], utxoValueBytesSlice)
-	fundedAt, _ := new(big.Int).SetString(defaultFundedAt, 0)
-
 	tlc.deposits[depositAddress] = &localDeposit{
 		keepAddress: keepAddress.Hex(),
 		state:       chain.AwaitingSignerSetup,
-		utxoValue:   big.NewInt(int64(chain.UtxoValueBytesToUint32(utxoValueBytes))),
 		fundingInfo: &chain.FundingInfo{
-			UtxoValueBytes:  utxoValueBytes,
-			FundedAt:        fundedAt,
-			TransactionHash: previousTransactionHashHex,
-			OutputIndex:     previousTransactionIndex,
+			FundedAt: big.NewInt(0),
```

Comment:
ahh, i see - `FundedAt` is probably not going to be <= 0 in normal cases, but it can definitely be set like that in tests/mocks

Reply:
Yes, we expect the lowest to be `0` but you never know...